Swiftification/NSAttributedString.swift

```diff
@@ -22,6 +22,26 @@
 
 import Foundation
 
+public extension NSAttributedString {
+ 
+    func substring(from range: Range<String.Index>) -> String {
+        return String(self.string[range])
+    }
+    
+    subscript(range: Range<String.Index>) -> String {
```

Comment:
I can take this out if you don't like it. I think its a little confusing, it probably should return an NSAttributedString, right?

Reply:
I think I'd go with returning an attributed string (you can then do `.string` on it to get the string), but if this method is useful to you I don't have a problem with it.

Reply:
Oh, the subscript... yeah, attributed string would be better